model/security/get_role_mappings/structures.smithy

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+structure GetRoleMappingsOutputStructure{
+    content: RoleMappings
+}
+
+@input
+structure GetRoleMappings_Input {}
+
+@output
+structure GetRoleMappings_Output {
+    @httpPayload
+    content: GetRoleMappingsOutputStructure
```

Comment:
This operation directly returns a map:
```suggestion
@input
structure GetRoleMappings_Input {}

@output
structure GetRoleMappings_Output {
    @httpPayload
    content: RoleMappings
```
It is expected that this will cause build errors until #127 is merged